src/handlers/get-priority-time.ts

```
@@ -0,0 +1,61 @@
+import { Decimal } from "decimal.js";
+
+export interface PriorityTimeEstimate {
+  time: string;
+  priority: string;
+}
+
+export interface PriorityTimeResponse {
+  estimated_time: string;
+  priority: string;
+}
+
+export async function getPriorityTime(
+  issueDescription: string,
+  issueTitle: string,
+  basetenApiKey: string,
+  basetenApiUrl: string = "https://model-yqvr8mjw.api.baseten.co/development/predict"
```

Do we want this string as a default? Feels like it makes more sense to make it required in the env.

---

Yes, I am saying that we could just throw an error if this is missing from the env as we usually do instead of having this default here, since there is already a way to set it in the env.